src/util/get-electron-version.js

```diff
@@ -0,0 +1,7 @@
+import path from 'path';
+import readPackageJSON from './read-package-json';
+
+export default async (projectDir) => {
+  const epcPackageJSON = await readPackageJSON(path.resolve(projectDir, 'node_modules', 'electron-prebuilt-compile'));
```

Comment:
Just call it `packageJSON`, we're going to [decouple `electron-compile` from Forge](https://github.com/electron-userland/electron-forge/pull/432).